backend/Origam.Schema.EntityModel/LookupServerSideElementRuleAttribute.cs

```diff
@@ -36,12 +36,16 @@ public override Exception CheckRule(object instance, string memberName)
         {
             if (memberName == String.Empty | memberName == null) CheckRule(instance);
             var iDataLookup = ((ILookupReference)instance).Lookup;
-            
+            if (iDataLookup == null)
```

Comment:
The rule should stop evaluating when lookup not set as the only thing the rule is supposed to check (by its name) is that the lookup is filtered server side.

The `Lookup` attribute should have `[NotNullModelElementRule]` to indicate that it is a mandatory value.